systemd/system-generators/coreos-autologin-generator

```diff
@@ -52,6 +64,8 @@ autologin() {
         overlay_unit "getty@.service"
         overlay_unit "serial-getty@.service"
     fi
+
+    emergency_autologin
```

Comment:
Note that multiple consoles may be listed but only the first is going to be applicable (only one can be stdin) so something like:

```sh
if [[ -n "$1" ]]; then
    egrep -q "^$1( |$)" /sys/class/tty/console/active || return
fi
```